test_requirements.txt

```diff
@@ -37,7 +37,7 @@ Pillow~=9.0
 plotly>=4.8.0, <6.0
 pre-commit>=2.9.2, <3.0  # The hook `mypy` requires pre-commit version 2.9.2.
 psutil==5.8.0
-pyarrow>=1.0, <7.0
+pyarrow>=1.0, <9.0
```

Comment:
Hopefully this doesn't break any other bits of Kedro!

Reply:
Hopefully not! But found this dependency update required for the snowpark df.to_pandas() method to work. 
Maybe it won't be required if _load() returns a snowpark df - but still could affect if it gets converted to pandas on a pipeline